docs/sources/alerting/_index.md

```diff
@@ -13,87 +13,54 @@ labels:
 menuTitle: Alerting
 title: Grafana Alerting
 weight: 114
+hero:
+  title: Grafana Alerting
+  level: 1
+  image: /media/docs/grafana-cloud/alerting-and-irm/grafana-icon-alerting.svg
+  width: 110
+  height: 110
+  description: Grafana Alerting allows you to learn about problems in your systems moments after they occur.
+cards:
+  title_class: pt-0 lh-1
+  items:
+    - title: Introduction
+      href: /docs/grafana-cloud/alerting-and-irm/alerting/
```

Comment:
The previous link linked to itself in Cloud and to the Cloud landing page from OSS. I think this is meant to go to the page with the first heading "Introduction". The relative path below works in both Grafana OSS and Cloud.
```suggestion
      href: ./fundamentals/
```

Reply:
hmm so i need all the links to work from both oss and cloud. should they all be changed?